tests/components/time_date/test_sensor.py

```diff
@@ -1,106 +1,83 @@
 """The tests for time_date sensor platform."""
 
 from freezegun.api import FrozenDateTimeFactory
+import pytest
 
 import homeassistant.components.time_date.sensor as time_date
 from homeassistant.core import HomeAssistant
 import homeassistant.util.dt as dt_util
 
+from . import load_int
+
 
 async def test_intervals(hass: HomeAssistant, freezer: FrozenDateTimeFactory) -> None:
     """Test timing intervals of sensors."""
-    device = time_date.TimeDateSensor(hass, "time")
+    device = time_date.TimeDateSensor("time", "1234567890")
     now = dt_util.utc_from_timestamp(45.5)
     freezer.move_to(now)
     next_time = device.get_next_interval()
-    assert next_time == dt_util.utc_from_timestamp(60)
-
-    device = time_date.TimeDateSensor(hass, "beat")
-    now = dt_util.parse_datetime("2020-11-13 00:00:29+01:00")
-    freezer.move_to(now)
-    next_time = device.get_next_interval()
-    assert next_time == dt_util.parse_datetime("2020-11-13 00:01:26.4+01:00")
+    assert next_time == dt_util.parse_datetime("1970-01-01 00:01:00+00:00")
 
-    device = time_date.TimeDateSensor(hass, "date_time")
+    device = time_date.TimeDateSensor("date_time", "1234567890")
```

Comment:
I'm assuming we'll get rid of the direct entity tests and move to tests via the state machine in a follow up PR?